blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpRollupsQueryHandler.java

```diff
@@ -141,13 +148,13 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
         } catch (InvalidRequestException e) {
             // let's not log the full exception, just the message.
             log.warn(e.getMessage());
```

Comment:
Here we are logging BAD_REQUEST as warnings.  Same question about if we want to log this, and at what level.